examples/preprocessing/plot_scaling_importance.py

```diff
@@ -137,3 +179,83 @@
 plt.tight_layout()
 
 plt.show()
+
+# %%
+# From the plot above we observe that scaling the features before reducing the
+# dimensionality results in components with the same order of magnitude. In this
+# case it also improves the separability of the clases. Indeed, in the next
+# section we confirm that a better separability has a good repercussion on the
+# overall model's performance.
+#
+# Effect of rescaling on model's performance
+# ==========================================
+#
+# First we show how the optimal regularization of a
+# class:`~sklearn.linear_model.LogisticRegressionCV` depends on the scaling or
+# non-scaling of the data:
+
+import numpy as np
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegressionCV
+
+Cs = np.logspace(-5, 5, 20)
+
+unscaled_clf = make_pipeline(pca, LogisticRegressionCV(Cs=Cs))
+unscaled_clf.fit(X_train, y_train)
+y_pred = unscaled_clf.predict(X_test)
+
+scaled_clf = make_pipeline(scaler, pca, LogisticRegressionCV(Cs=Cs))
+scaled_clf.fit(X_train, y_train)
+y_pred_scaled = scaled_clf.predict(X_test)
+
+print("Optimal C for the unscaled PCA")
+print(unscaled_clf[-1].C_)
+print()
+print("Optimal C for the standardized data with PCA")
+print(scaled_clf[-1].C_)
+
+# %%
+# The need for regularization is higher (lower values of `C`) for the data
+# that was not scaled before applying PCA. From the plot we can confirm that
```

Comment:
By plotting the validation curves I realized that the training and test accuracy overlap too much to make a proper statement about over- or underfitting for the scenario with no standardization.

![output1](https://user-images.githubusercontent.com/86408019/212720612-281ee478-d600-408e-b06a-2c2d8fc25400.png)
![output2](https://user-images.githubusercontent.com/86408019/212720626-10690ddf-0368-4528-9f72-23b6fa95b6a1.png)

I think that it is better to avoid mentioning over-/underfitting to keep the example as simple as possible.